### DISCUSSION TITLE
Issue installing on Nextjs

### DISCUSSION THREAD

**Discussion by ahmedsamirdev** [zeit-ui/react#540] [Q&A] [answered]
Whenever I try to follow the example u gave I got this error ..
![unknown](https://user-images.githubusercontent.com/16477954/118318771-21fd3c80-b4fa-11eb-95f3-e48cf6854eb6.png)

 I followed exactly the example here .. but nothing works
 https://github.com/geist-org/react/tree/master/examples/create-next-app

**Comment by unix** [comment, marked as answer]
Just upgrade the deps version.

**Reply by unix** [reply]
If the problem cannot be solved, you can create an online project (Codesandbox or GitHub CodeSpaces) and I will follow up on the issue.

**Reply by ahmedsamirdev** [reply]
https://codesandbox.io/s/distracted-matsumoto-hur1w?file=/pages/index.js
yeah, it works online.. what could be the issue with my pc .. as the error says it could be hooks!!
